datahub-web-react/src/app/ingest/source/builder/sources.json

```diff
@@ -48,6 +48,13 @@
         "docsUrl": "https://datahubproject.io/docs/generated/ingestion/sources/tableau/",
         "recipe": "source:\n  type: tableau\n  config:\n    # Coordinates\n    connect_uri: null\n        stateful_ingestion:\n            enabled: true"
     },
+    {
+        "urn": "urn:li:dataPlatform:dbt",
+        "name": "dbt-cloud",
```

Comment:
I believe `urn:li:dataPlatform:dbt` is shared between dbt and dbt Cloud, so this shouldn't be specific to dbt-cloud

Reply:
it is not. you can have multiple entries with same urn